lib/pbench/server/database/models/datasets.py

```diff
@@ -270,10 +270,7 @@ class Dataset(Database.Base):
         access      Dataset is "private" to owner, or "public"
         name        Base name of dataset (tarball)
         md5         The dataset MD5 hash (Elasticsearch ID)
-        created     Tarball metadata timestamp (set during PUT)
         uploaded    Dataset record creation timestamp
```

Comment:
Rename `md5` to `resource_id`?

(And, do we want to say anything about the two `relationship`'s?)

Reply:
Ha; I just happened to notice that I'd neglected to remove the obsolete columns, but I didn't notice the (much earlier) "spelling change".

I've debated several times changing the header from `Columns` to the more conventional `Args`, which would leave room for non-column attributes like the relationships ... which indeed can be specified as arguments. But in *this* format, they don't really fit.